templates/lobby.html

```diff
@@ -726,6 +737,7 @@
         } else if (parsed.type === "close-guess") {
             applyMessage("close-guess-message", "System", "'"+ parsed.data + "'' is very close.")
         } else if (parsed.type === "update-wordhint") {
+            waitingDialog.style.visibility = "hidden";
```

Comment:
A bit weird, since it's not obvious that this event necessarily implies that a word has been chosen, maybe we can add a comment with our assumption here. Or make the eventing a bit more concise, by making a "word has been chosen" event.